documentation/src/components/side-nav.css

```diff
@@ -14,11 +14,14 @@ governing permissions and limitations under the License.
     background-color: var(--spectrum-global-color-gray-100);
 }
 
+aside {
```

Comment:
Discovered that the responsive change I made prevented the navigation content from scrolling...